storage/cache/log_subtree_cache.go

```diff
@@ -35,6 +35,49 @@ func LogPopulateFunc(hasher hashers.LogHasher) storage.PopulateSubtreeFunc {
 	return populateLogSubtreeNodes(hasher)
 }
 
+func suffixKey(d, i int) string {
```

Comment:
I can add this to my Suffix rework perhaps? Or do you think the logic belongs better here?